internal/connector/pingone/resources/sso/pingone_application_attribute_mapping.go

```diff
@@ -0,0 +1,96 @@
+package sso
+
+import (
+	"fmt"
+
+	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/connector/pingone/resources/common"
+	"github.com/pingidentity/pingctl/internal/logger"
+)
+
+// Verify that the resource satisfies the exportable resource interface
+var (
+	_ connector.ExportableResource = &PingoneApplicationAttributeMappingResource{}
+)
+
+type PingoneApplicationAttributeMappingResource struct {
+	clientInfo *connector.SDKClientInfo
+}
+
+// Utility method for creating a PingoneApplicationAttributeMappingResource
+func ApplicationAttributeMapping(clientInfo *connector.SDKClientInfo) *PingoneApplicationAttributeMappingResource {
+	return &PingoneApplicationAttributeMappingResource{
+		clientInfo: clientInfo,
+	}
+}
+
+func (r *PingoneApplicationAttributeMappingResource) ExportAll() (*[]connector.ImportBlock, error) {
+	l := logger.Get()
+
+	l.Debug().Msgf("Fetching all %s resources...", r.ResourceType())
+
+	apiExecuteApplicationsFunc := r.clientInfo.ApiClient.ManagementAPIClient.ApplicationsApi.ReadAllApplications(r.clientInfo.Context, r.clientInfo.ExportEnvironmentID).Execute
+	apiApplicationFunctionName := "ReadAllApplications"
+
+	embedded, err := common.GetManagementEmbedded(apiExecuteApplicationsFunc, apiApplicationFunctionName, r.ResourceType())
+	if err != nil {
+		return nil, err
+	}
+
+	importBlocks := []connector.ImportBlock{}
+
+	l.Debug().Msgf("Generating Import Blocks for all %s resources...", r.ResourceType())
+
+	for _, app := range embedded.GetApplications() {
+		var (
+			appId     *string
+			appIdOk   bool
+			appName   *string
+			appNameOk bool
+		)
+
+		switch {
+		case app.ApplicationOIDC != nil:
+			appId, appIdOk = app.ApplicationOIDC.GetIdOk()
+			appName, appNameOk = app.ApplicationOIDC.GetNameOk()
+		case app.ApplicationSAML != nil:
+			appId, appIdOk = app.ApplicationSAML.GetIdOk()
+			appName, appNameOk = app.ApplicationSAML.GetNameOk()
+		default:
```

Comment:
looks like you're missing a case for app.ApplicationExternalLink here

Reply:
See the pingone provider resource logic
https://github.com/pingidentity/terraform-provider-pingone/blob/e169edbe77031bc1e72309d77241a9776cc1566c/internal/service/sso/resource_application.go#L734

Reply:
Added WSFED